### PR TITLE
AI-86: Fix merge conflict

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
## Before
filter-acivity

table-pagination

## After
resolved-filter-acivity

resolved-table-pagination

---

- [x] Tests Pass

### CODE REVIEW

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss [hunk 1]
```diff
@@ -375,180 +373,237 @@
...
+          text-overflow: ellipsis;
+          vertical-align: middle;
+          white-space: nowrap;
+
+          a {
+            color: $brand-primary !important;
+          }
+        }
+
+        td:nth-child(3), td:nth-child(4) {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss]: Please separate each selector in a separate line as you did in line 422 above 
```
.dataTables_info,
.dataTables_paginate {
```

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss [hunk 2]
```diff
@@ -375,180 +373,237 @@
...
+      &:before {
+        bottom: 2px;
+        font-size: $font-size-base !important;
+        position: relative;        
+      }
+    }
+      
+    .crm-icon.Individual-icon {
+      background: none;
+      &:before {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss]: Add empty line above

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss [hunk 3]
```diff
@@ -375,180 +373,237 @@
...
 
-        &.Organization-icon {
-          background: none;
-          &:before {
-            @include fa-icon($content: "\f0f7");
-          }
-        }
+    .crm-icon.Organization-icon {
+      background: none;
+      &:before {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss]: Add empty line above

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss [hunk 4]
```diff
@@ -375,180 +373,237 @@
...
 
-        &.Household-icon {
-          background: none;
-          &:before {
-            @include fa-icon($content: "\f015");
-          }
-        }
+    .crm-icon.Household-icon {
+      background: none;
+      &:before {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss]: Add empty line above

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss [hunk 5]
```diff
@@ -375,180 +373,237 @@
...
-          border-right: none;
-
-          td {
-            background: $crm-white;
-            border: none;
-            color: $gray-dark;
-            line-height: 18px;
+          border: none;
+          color: $gray-dark;
+          line-height: 18px;
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss]: Can you use em instead for line-height?

**Comment by mukeshcompucorp** [comment]
Resolved line-height issue
Resolved Add empty line above issues
Resolved separate each selector in a separate line issue

**Pull request merged**